components/Pages/DocuSignPage/CartTotal.tsx

```diff
@@ -32,59 +32,51 @@ const CartTotal = () => {
           </h2>
           <hr className="border border-grey -mt-1" />
         </div>
-        <div className="flex flex-col gap-2 text-left font-share place-self-center">
+        <div className="flex flex-col gap-2 text-left font-share place-self-start">
           <div className="w-full flex justify-between gap-2 md:text-xl">
             <p>1x</p>
-            <p>Automized Contract</p>
+            <p className="md:text-[24px]">Automized Contract</p>
           </div>
         </div>
 
         <div className="flex flex-col gap-2 font-share text-[11px]">
           <div className="w-full flex items-center gap-2 ">
-            <p>Sub-Total</p>
+            <p className="md:text-[18px]">Sub-Total</p>
```

Comment:
Ensure that all texts (e.g., "Sub-Total," "Taxes") are localized, allowing for easier future translation or adaptation.